src/System.Private.CoreLib/shared/System/Security/IPermission.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
There appear to be missing annotations in this file, e.g.
- Intersect can both accept and return null
- IsSubsetOf can accept null
- Union can both accept and return null

Reply:
I couldn't find doc say anything that null is fine (although it is handled by example) - I only found something about `*Impl` versions of the API that they should return null when null is given.

will change these

Reply:
I also looked at netfx's implementations.